src/gui/mzroll/mainwindow.cpp

```diff
@@ -3249,42 +3239,30 @@ QWidget* MainWindowWidgetAction::createWidget(QWidget *parent) {
 
 		QToolButton *btnNext = new QToolButton(parent);
 		btnNext->setIcon(QIcon(rsrcPath + "/next.png"));
-		btnNext->setToolButtonStyle(Qt::ToolButtonTextUnderIcon);
 		btnNext->setToolTip(tr("History Forward (Ctrl+Right)"));
 		btnNext->setShortcut(tr("Ctrl+Right"));
 		connect(btnNext, SIGNAL(clicked()), mw, SLOT(historyNext()));		
 		return btnNext;
 
 	}
-	else if (btnName == "btnPDF") {
-
-		QToolButton *btnPDF = new QToolButton(parent);
-		btnPDF->setIcon(QIcon(rsrcPath + "/exportpdf.png"));
-		btnPDF->setToolButtonStyle(Qt::ToolButtonTextUnderIcon);
-		btnPDF->setToolTip(tr("Save EIC Image to PDF file"));
-		connect(btnPDF, SIGNAL(clicked()), mw, SLOT(exportPDF()));
-		return btnPDF;
-
-	}
-	else if (btnName == "btnPNG") {
-
-		QToolButton *btnPNG = new QToolButton(parent);
-		btnPNG->setIcon(QIcon(rsrcPath + "/copyPNG.png"));
-		btnPNG->setToolButtonStyle(Qt::ToolButtonTextUnderIcon);
-		btnPNG->setToolTip(tr("Copy EIC Image to Clipboard"));
-		connect(btnPNG, SIGNAL(clicked()), mw, SLOT(exportSVG()));
-		return btnPNG;
-
-	}
-	else if (btnName == "btnPrint") {
-
-		QToolButton *btnPrint = new QToolButton(parent);
-		btnPrint->setIcon(QIcon(rsrcPath + "/fileprint.png"));
-		btnPrint->setToolButtonStyle(Qt::ToolButtonTextUnderIcon);
-		btnPrint->setToolTip(tr("Print EIC (Ctr+P)"));
-		btnPrint->setShortcut(tr("Ctrl+P"));
-		connect(btnPrint, SIGNAL(clicked()), mw, SLOT(print()));
-		return btnPrint;
+	else if (btnName == "btnExport") {
+		
+		QToolButton* btnExport = new QToolButton(parent);
+		btnExport->setIcon(QIcon(rsrcPath + "/exportOptions.png"));
+		btnExport->setToolTip("Export as…");
+		QMenu* btnExportMenu = new QMenu("Export Image as…");
+		btnExport->setMenu(btnExportMenu);
+		btnExport->setPopupMode(QToolButton::InstantPopup);
+		QAction* pdfAction = btnExportMenu->addAction(QIcon(rsrcPath + "/exportpdf.png"),
+													  "Export EIC as PDF file");
+		connect(pdfAction, SIGNAL(triggered(bool)), mw, SLOT(exportPDF()));
```

Comment:
lets try to use the new  syntax of signals and slots wherever possible.  In Qt5 the same will look something like this  `connect(pdfAction, &QAction::triggered, mw, &MainWindow::exportPdf)`. 

One of the pros is that compile-time check is enabled for the new syntax. Read more [here](https://wiki.qt.io/New_Signal_Slot_Syntax)

Reply:
Done.